corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -357,7 +357,7 @@ def __unicode__(self):
         return "%s synced on %s (%s)" % (self.user_id, self.date.date(), self.get_id)
 
 
-class OwnershipCleanliness(models.Model):
+class OwnershipCleanlinessFlag(models.Model):
```

Comment:
I'm doing something a bit sketchy though quite convenient here.

This database table is completely empty on all of our servers (logic was such that you would never save to it unless something was already there which was dependent on this PR for happening).

At the same time, I forgot to check in the initial migration with the original model.

By renaming the model (to a more appropriate name, i'll add) I get to add a migration file at the same time AND change the schema AND not have to worry about running any manual fake migrations on prod.

The downside is there will be an empty db table hanging around for the old model, but I'll go clean those out after this gets merged and deployed.